src/main.rs

```
@@ -1,34 +1,24 @@
 use tracing::{error, info};
 
-mod redis_lib;
-mod mysql_lib;
 mod bot;
+mod mysql_lib;
+mod redis_lib;
 
 #[tokio::main]
 async fn main() {
     tracing_subscriber::fmt::init();
 
-    let redis_client;
-    match redis_lib::get_connection() {
-        None => return,
-        Some(val) => redis_client = val
-    }
+    let redis = redis_lib::get_connection().await.unwrap();
     info!("Successfully connected to redis");
 
-    let sql_pool;
-    match mysql_lib::get_connection(5).await {
-        None => return,
-        Some(val) => sql_pool = val
-    }
+    let sql_pool = mysql_lib::get_connection(5).await.unwrap();
```

Handle `None` and don't let the function panic by the use of `unwrap`

---

after reading more, panicking in the early stages of the main function is fine.